src/main/docs/guide/kubernetes-client-openapi/kubernetes-client-openapi-config-client.adoc

```diff
@@ -287,6 +299,17 @@ kubernetes:
         - "app.kubernetes.io/instance"
 ----
 
+Similarly to ``ConfigMap``s, in case if you want to terminate the service startup when the service is not able to read secrets
+because of any unexpected exception, you can enable `terminate-startup-on-exception` property:
```

Comment:
```suggestion
Similar to ``ConfigMap``s, if you want to terminate the service startup when the service is not able to read secrets
because of any unexpected exception, you can set `terminate-startup-on-exception` property to `true` in application configuration:
```

Reply:
done